### ISSUE TITLE
Use PHP mail with system?

### ISSUE THREAD

**Issue by cogatserofulous** [grokability/snipe-it#9646]
Hi,
how to set up mail to use the default php mail of the server and not SMTP.
I've set up numerous other scripts that send email and they send emails straight away using server's built in php mail w/o any additional configuration.
I can't figure out how to set up snipe-it to do this...I see that you do have an option for "mail" and "sendmail" though, in .env and mail.php.
What setting changes and where to get this working?
Thanks!

**Comment by uberbrady** [comment]
In your `.env` file you need to set your `MAIL_DRIVER` to `mail` which should be using PHP's mail function. You may also want to try `sendmail` (as you mentioned) but that will only work if you have sendmail living at `/usr/sbin/sendmail`.

If mail isn't working, definitely check your Laravel logs to see what's going wrong (typically at `storage/logs/laravel.log`)

**Comment by cogatserofulous** [comment]
Thanks. I'm getting this: "production.ERROR: Driver [php] not supported. {"exception":"[object] (InvalidArgumentException(code: 0): Driver [php] not supported"
But I just installed another script that sends mail and it started sending mail immediately. The server is working with php mail then, correct?

**Comment by uberbrady** [comment]
The name of the driver is `mail`, not `php`. So you should have:

`MAIL_DRIVER=mail`

In your `.env` file. Yes, if another script was able to send emails right away without configuring an SMTP server, then your server _should_ be able to send mail directly.

If you keep getting the same error after that, you may have a configuration caching issue, try running `php artisan config:clear` which should clear that out.

If it still doesn't work, maybe something else is up - try and paste your .env file here (make sure to redact out any passwords, of course). Maybe something else is wrong in there?

**Comment by cogatserofulous** [comment]
yes mail is used in that spot. I guess some other obvious questions would be am I leaving the other field with "null" or am i filling out, for example, The Host field, the username password fields, the encryption field etc or do I leave them blank because that is if I am using SMTP only? thanks!

**Comment by uberbrady** [comment]
you can delete those lines entirely. Still weird it talks about a 'php' driver, that doesn't make sense. I would try the `php artisan config:clear`

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
